tests/AutoReview/ProjectCodeTest.php

```diff
@@ -339,6 +339,7 @@ private function getTestClasses()
             ->in(__DIR__.'/..')
             ->exclude(array(
                 'Fixtures',
+                'Test',
```

Comment:
having them in `src` requires you to have whole dev-deps in `require` instead of `dev-require`. we face it with GeckoPackages and PHPUnit packages. that's what composer requirements checker complains about. so current pathing is broken